app/views/examples/branching.html

```diff
@@ -21,15 +19,63 @@ <h1 class="heading-xlarge">
         <p>
           You may want to show a user different pages, depending on how they've answered a question.
         </p>
-
+        
         <p>
-          You can see the code for this example here:
+          You can make this happen by including <code class="code">value="redirect(&lt;URL&gt;)"</code> in your form inputs. For example:</URL>
         </p>
 
+        <div class="code" style="width:110%">
+  &lt;input type=&quot;radio&quot; name=&quot;work-status&quot; value=&quot;redirect(branch/employed)&quot;&gt;
+  &lt;input type=&quot;radio&quot; name=&quot;work-status&quot; value=&quot;redirect(branch/selfemployed)&quot;&gt;
+  &lt;input type=&quot;radio&quot; name=&quot;work-status&quot; value=&quot;redirect(branch/director)&quot;&gt;
+        </div>
+        
+        <form action="/examples/branching" method="get" class="form">
+                  
+          <div class="form-group">
+            
+            <h2 class="heading-medium">
+              What is your employment status?
+            </h2>          
+            
+            <fieldset class="">
+
+              <legend class="visuallyhidden">What is your employment status?</legend>
+
+              <label for="workStatusEmp" class="block-label">
+                <input type="radio" name="work-status" value="redirect(branch/employed)" id="workStatusEmp">
+                Employed
+              </label>
+
+              <label for="workStatusSelf" class="block-label">
+                <input type="radio" name="work-status" value="redirect(branch/selfemployed)" id="workStatusSelf">
+                Self-employed
+              </label>
+              
+              <label for="workStatusDir" class="block-label">
+                <input type="radio" name="work-status" value="redirect(branch/director)" id="workStatusDir">
+                Director of a limited company
+              </label>
+
+            </fieldset>            
+          </div>
+          
+          <div class="form-group">
+            <input type="submit" class="button" value="Continue">
+          </div>
+        
+        </form>
+        
+        <p>&nbsp;</p>
+        
+        <p>If you need more comple logic to determine which page to show next you can write server-side code to achieve this as in the following example:</p>
```

Comment:
I don't think this paragraph gives enough information. It references the following example, but they are locations of files. The previous text made it more explicit that they were references to where the code could be found.

Alternatively, we could include some of the branching logic in the page and tell users the full logic is in x file.